### DISCUSSION TITLE
Stock Data Download Issue

### DISCUSSION THREAD

**Discussion by duvvurirao64** [pranjal-joshi/Screeni-py#143] [Q&A] [answered]
after running the screenipy exe file first day i got list of stocks in the table form, but second day onwards showing zero stocks in the table. requesting to help me to solve this problem
thanks in advance
![image](https://user-images.githubusercontent.com/120102901/206442833-7a545858-6c6b-4b30-9bfb-d4faf1b171bc.png)

**Comment by pranjal-joshi** [comment]
Hi @duvvurirao64 

Please try if it works for `All Stocks` rather than Nifty 200.
Without any error log, it is difficult to debug.

**Comment by pranjal-joshi** [comment, marked as answer]
Hi @duvvurirao64 

I've created a new issue #144 for debugging and resolving this problem.
Locking this conversation as it should be an issue and not a Q&A.

Please follow this to communicate further #144

**Comment by pranjal-joshi** [comment]
Fixed this in #144
